app/staticdata/categories.json

```diff
@@ -0,0 +1,37 @@
+[
```

Comment:
I think we should have an option to upload photos, because I wouldn't want to keep all these images in repo, but we still should keep them together with the app.